### ISSUE TITLE
[Accessibility] Green and yellow score text is low contrast

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [ampproject/amp.dev#4501]
Environment:
Any OS/browser

Steps to reproduce:
1. Open https://amp-dev-staging.appspot.com/page-experience/
2. Enter a valid URL, that will produce a range of scores, and press Analyze

Observe the green and yellow score text

"Good" and accompanying time in green
#00CD3C
Contrast ratio: 2.14:1

"Needs improvement" and accompanying time in yellow
#F39C12
Contrast ratio: 2.19:1

Both of these contrast ratios should be at least 4.5:1. See [WCAG 1.4.3](https://www.w3.org/TR/WCAG21/#contrast-minimum) and [Understanding Success Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
![image](https://user-images.githubusercontent.com/30604285/92966806-0acaf400-f42d-11ea-89e8-42afc080c9f5.png)

**Comment by nainar** [comment]
Suggested colour replacement: 
Good - #178739 (4.6:1 contrast)

**Comment by caroqliu** [comment]
@PeterSaathoff-Harshfield Would 3:1 contrast for the orange be acceptable? The darker version of yellow moves towards brown (undesirable) and if we warm it up to make it more reasonable, it becomes too red and therefore easier to confuse with the "Poor" result.

**Comment by nainar** [comment]
The closest I can get for Needs Improvement is #E26E08 - 3.24:1

**Comment by caroqliu** [comment]
I noticed the headline is 28pt and the score text (3.3s and 68ms in the above examples) is 16pt bold. It looks like this is considered [large scale text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-large-scale), so we will instead update the contrast to 3:1 instead of 4.5:1 as recommended in the original issue.

**Comment by PeterSaathoff-Harshfield** [comment]
Looks good to me

**Comment by caroqliu** [comment]
Thank you!

**Comment by caroqliu** [comment]
Note also that these color values were updated in #4579 and still meet the 3:1 contrast as discussed above.
